sql/core/src/main/java/org/apache/spark/sql/sources/v2/Table.java

```diff
@@ -18,9 +18,6 @@
 package org.apache.spark.sql.sources.v2;
 
 import org.apache.spark.annotation.Evolving;
-import org.apache.spark.sql.sources.v2.reader.Scan;
-import org.apache.spark.sql.sources.v2.reader.ScanBuilder;
-import org.apache.spark.sql.types.StructType;
 
 /**
  * An interface representing a logical structured data set of a data source. For example, the
```

Comment:
According to this update, maybe `a logical structured data set` -> `a logical named data set`?